.github/workflows/build.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       ROX_PRODUCT_BRANDING: ${{ matrix.branding }}
     runs-on: ubuntu-latest
     container:
-      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.55
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.57.1
```

Comment:
Why version has `.1` suffix? Why not usual x.y.z?

Reply:
We branched out from 0.3.57 to create an image with 1.19.9, >0.3.58 are on Go 1.20 already.